torch/nn/modules/container.py

```diff
@@ -318,6 +318,18 @@ def _get_abs_string_index(self, idx):
             idx += len(self)
         return str(idx)
 
+    @overload
+    def __getitem__(self, idx: slice) -> "ModuleList":
+        ...
+
+    @overload
+    def __getitem__(self, idx: int) -> Module:
+        ...
+
+    @overload
+    def __getitem__(self, idx: Union[int, slice]) -> Union[Module, "ModuleList"]:
+        ...
```

Comment:
```suggestion
 ```
 I think this can be removed unless the decorator below it is messing up the type checker. The typing on `__getitem__` should cover it?

Reply:
you are right